charts/httpd/templates/httpdRestart.yaml

```
@@ -0,0 +1,64 @@
+{{- if .Values.httpdRestart.enable }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+rules:
+  - apiGroups: ["apps", "extensions"]
+    resources: ["deployments"]
+    resourceNames: [{{ include "httpd.fullname" . }}]
+    verbs: ["get", "patch", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "httpd.fullname" . }}-restart-deployment
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-cronjob
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 2
+  concurrencyPolicy: Forbid
+  {{- with .Values.httpdRestart.cron }}
+  schedule: {{ toYaml . }}
+  {{- end }}
```

```suggestion
  schedule: {{ .Values.httpdRestart.cron }}
```

- The `.Values.httpdRestart.cron` is not yaml: we have to remove the `toYaml` template function
- Using the `with` structure is not useful here: I assume `schedule` keyword is mandatory and must be specified.